versioned_docs/version-23.10/administration/centreon-ha/acceptance-guide.md

```diff
@@ -1147,7 +1147,7 @@ The @CENTRAL_SLAVE_NAME@ will be completely excluded from the cluster. The @CENT
 
 ### Processing 
 
-To perform this test, launch the `iptables` commands on the @CENTRAL_SLAVE_NAME@. Be carreful avoid @CENTRAL_SLAVE_IPADDR@ otherwise you loose SSH connection on this node.
+To perform this test, launch the `iptables` commands on the @CENTRAL_SLAVE_NAME@. Be careful to avoid @CENTRAL_SLAVE_IPADDR@, otherwise you lose the SSH connection on this node.
```

Comment:
```suggestion
To perform this test, launch the `iptables` commands on the @CENTRAL_SLAVE_NAME@. Be careful to not create iptables rules containing @CENTRAL_SLAVE_IPADDR@, otherwise you lose the SSH connection on this node.
```